### ISSUE TITLE
Consistent naming, consider changing from IceSat2 to ICESat-2

### ISSUE THREAD

**Issue by evetion** [mlampros/IceSat2R#8]
Your package is called `IceSat2R`, but the official name of the mission is `ICESat-2`. Note the capitals and dash. I'm not sure whether the package can be renamed (`ICESat2R?`), but I think the documentation could use consistent naming of ICESat-2 in the titles.

**Comment by mlampros** [comment]
Yes you might have right regarding the name of the R package. I used the same parameter names as is the case in OpenAltimetry but I didn't do the same for the package name. Now that I've already submitted the package to CRAN I think it won't be wise to rename the package due to the existing R users. I'll have a look to the documentation of the package once again in the weekend and I'll adjust the cases where the naming is not consistent.
